draft-ietf-mmusic-dtls-sdp.xml

```diff
@@ -99,11 +99,14 @@
                 or answers can be used to indicate whether a new DTLS association is to be established.
             </t>
             <t>
-                The SDP 'tls-id' attribute can also be used for negotiating a TLS connection, using
+                The SDP 'tls-id' attribute can be specified when negotiating a TLS connection, using
                 the procedures in this document in conjunction with the procedures in <xref format="default"
                 pageno="false" target="RFC5763"/> and <xref format="default" pageno="false" target="RFC8122"/>.
-                The TLS specific considerations are described in <xref format="default" pageno="false"
-                target="sec-tls-cons"/>.
+                The unique combination of SDP 'tls-id' attribute values can be used to identity the negotiated

```

Comment:
Both the offerer and answerer will provide a tls-id value, and the TLS connection will be identified using the combination of those values.